src/tyr/serializers.cc

```diff
@@ -156,6 +156,45 @@ void openlr(const valhalla::Api& api, int route_index, rapidjson::writer_wrapper
   }
   writer.end_array();
 }
+
+std::string serializePbf(Api& request) {
```

Comment:
this is the new function for serailizing pbf. basically the only tricky part is, if its a service request, we need to keep the options bits attached to the object until we are done processing the request, so we swap it out before we serialize rather than actually clearing it and then we swap it back in after we have the bytes we want to return.